alacritty/src/display/mod.rs

```diff
@@ -1061,9 +1061,11 @@ impl Display {
     #[inline]
     fn damage_highlighted_hints<T: EventListener>(&self, terminal: &mut Term<T>) {
         let display_offset = terminal.grid().display_offset();
+        let last_line = terminal.topmost_line();
```

Comment:
```suggestion
        let topmost_line = terminal.topmost_line();
```

Reply:
Thta's not topmost line, topmost is in the grid history, but I want the viewport one.